python/cmdLineUtils/CMakeLists.txt

```diff
@@ -20,351 +20,358 @@ ROOTTEST_ADD_TEST(SimplePattern3
                   OUTREF SimplePattern3.ref)
 #########################################################################
 
-set(TOOLS_PREFIX ${ROOTSYS}/bin)
+if(MSVC)
+    # the command line tools works fine in the Windows command prompt but
+    # not from CTest, so let's add Python.exe and the .py file extension
+    set(TOOLS_PREFIX ${PYTHON_EXECUTABLE} ${ROOTSYS}/bin)
```

Comment:
You should use `${PYTHON_EXECUTABLE_Development_Main}` instead of `${PYTHON_EXECUTABLE}`, since the former will always point to the higher Python for which a development package has been found, and therefore that is the one that PyROOT tests should run with (the command line utils also use PyROOT). If you use `${PYTHON_EXECUTABLE}`, it can happen that it points to a Python that does not have development files (just the interpreter), which is fine for ROOT, but not for PyROOT.

Reply:
@etejedor  ${PYTHON_EXECUTABLE_Development_Main} is somehow not set, so I had  to set it [there](https://github.com/root-project/roottest/pull/554/commits/0a2465917ab179b71c0c02b3da468b13e8da2a1e)

Reply:
If should always be set if CMake finds at least one Python installation that has the development package. It is set here:

https://github.com/root-project/root/blob/master/cmake/modules/SearchRootCoreDeps.cmake

If it is not set we should find out why, all the Python tests should run with the Python that `${PYTHON_EXECUTABLE_Development_Main}` points to.

Reply:
OK, thanks, I already found it in ROOT, but how is it set in roottest? Is it supposed to somehow inherit it?

Reply:
..and is it set if roottest is built independently of ROOT? :)

Reply:
> ..and is it set if roottest is built independently of ROOT? :)

What do you mean? `roottest` built without `ROOT`?

Reply:
@oshadura might know more

Reply:
> > ..and is it set if roottest is built independently of ROOT? :)
> 
> What do you mean? `roottest` built without `ROOT`?

roottest built independently of root, against a pre-installed root

Reply:
Looks like we need to sync some part of https://github.com/root-project/root/blob/master/cmake/modules/SearchRootCoreDeps.cmake here https://github.com/root-project/roottest/blob/master/CMakeLists.txt#L136 to make standalone roottest working properly.

Reply:
> Looks like we need to sync some part of https://github.com/root-project/root/blob/master/cmake/modules/SearchRootCoreDeps.cmake here https://github.com/root-project/roottest/blob/master/CMakeLists.txt#L136 to make standalone roottest working properly.

Thanks @oshadura !

Reply:
> > > ..and is it set if roottest is built independently of ROOT? :)
> > 
> > 
> > What do you mean? `roottest` built without `ROOT`?
> 
> roottest built independently of root, against a pre-installed root

Yes, and that's the issue. It is now broken, as it's the way I tested the PR on Windows